gear-programs/vft-treasury/src/wasm/tests/utils_gclient.rs

```diff
@@ -0,0 +1,226 @@
+/* gclient utils */
+use gclient::EventProcessor;
+use gclient::{EventListener, GearApi, Result};
+use gear_core::ids::*;
+use sails_rs::{Decode, Encode};
+use sails_rs::{H160, U256};
+use vft_treasury_app::services::{Config, InitConfig};
+pub async fn common_upload_program(
```

Comment:
As I see you don't use it anywhere except `upload_program`, so I think it can be only 1 function